fixtures/keywords/kotlin/src/lib.rs

```diff
@@ -15,9 +15,9 @@ impl r#break {
 }
 
 #[allow(non_camel_case_types)]
-trait r#continue {
+trait r#continue: Send + Sync {
     fn r#return(&self, v: r#return) -> r#return;
-    fn r#continue(&self) -> Option<Box<dyn r#continue>>;
+    fn r#continue(&self) -> Option<Arc<dyn r#continue>>;
```

Comment:
I don't understand why these changes are necessary - it implies something might have changed that we should document, but I'm really not clear why this is the only fixture impacted!